nautobot/dcim/templates/dcim/virtualdevicecontext_retrieve.html

```diff
@@ -65,4 +65,5 @@
 
 {% block content_full_width_page %}
     {% include 'panel_table.html' with table=interfaces_table heading="Interfaces" %}
+    {% include 'panel_table.html' with table=vrf_table heading="Assigned VRFs" %}
```

Comment:
How much work would it be to convert this view to the UI framework while you're in here?

Reply:
I refactored it here. But I wonder if we should leave this for the QA team in the future.